docs/src/main/asciidoc/micrometer.adoc

```diff
@@ -398,6 +398,11 @@ string. For example, setting
 `quarkus.micrometer.binder.http-server.match-patterns=/example/prime/[0-9]+=/example/{jellybeans}` would use the value
 `/example/{jellybeans}` for the uri attribute any time the requested uri matches `/example/prime/[0-9]+`.
 
+.Exported metrics format
+
+By default, the  metrics are exported using micrometer `application/openmetrics-text`, you can revert to the former
+format by specifying the `Accept` header request to `plain/text`.
```

Comment:
```suggestion
By default, the  metrics are exported using the Micrometer format `application/openmetrics-text`,
you can revert to the former format by specifying the `Accept` request header to `plain/text`.
```

Reply:
Used your suggestions and added a curl exemple

Reply:
hmm. These are not micrometer formats.. they are defined by Prometheus. You're choosing between Prometheus-defined formats (Micrometer itself has nothing to do with it). Does that matter?

Reply:
Amended accordingly to @ebullient hint